### PR TITLE
Add native USDC to ARB Pool - review

### PR DESCRIPTION
**Pull request by marczeller**
- payload (using custom listing engine to avoid version issue)
- makefile Update
- test (fails Need fix)
- AIP.md file

### CODE REVIEW

File: src/AaveV3ArbNativeUSDCListing_20230621/AaveV3ArbNativeUSDCListing_20230621.sol [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          variableRateSlope2: _bpsToRay(60_00),
+          stableRateSlope1: _bpsToRay(3_50),
+          stableRateSlope2: _bpsToRay(60_00),
+          baseStableRateOffset: _bpsToRay(1_00),
+          stableRateExcessOffset: _bpsToRay(8_00),
+          optimalStableToTotalDebtRatio: _bpsToRay(20_00)
+        }),
+        enabledToBorrow: EngineFlags.ENABLED,
+        stableRateModeEnabled: EngineFlags.DISABLED,
+        borrowableInIsolation: EngineFlags.DISABLED,
```
Comment (sakulstra) [inline src/AaveV3ArbNativeUSDCListing_20230621/AaveV3ArbNativeUSDCListing_20230621.sol]: should be enabled

File: src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          variableRateSlope2: _bpsToRay(60_00),
+          stableRateSlope1: _bpsToRay(3_50),
+          stableRateSlope2: _bpsToRay(60_00),
+          baseStableRateOffset: _bpsToRay(1_00),
+          stableRateExcessOffset: _bpsToRay(8_00),
+          optimalStableToTotalDebtRatio: _bpsToRay(20_00)
+        }),
+        enabledToBorrow: EngineFlags.ENABLED,
+        stableRateModeEnabled: EngineFlags.DISABLED,
+        borrowableInIsolation: EngineFlags.DISABLED,
```
Comment (sakulstra) [inline src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md:67]: should be enabled

File: src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+---
+
+## Simple Summary
+
+This AIP propose to add the native USDC token issued by Circle to the Arbitrum V3 pool.
+Currently onboarded USDC.e token will be kept as is. and two USDC tokens will be available on the pool. the bridged asset USDC.e and the native USDC with the symbol USDCn.
+
+## Motivation
+
+Circle is set to launch a native version of USDC on Arbitrum, which is anticipated to become the primary version of the USDC stablecoin on the Arbitrum L2. This will supersede the current “USDC.e” token, which is a bridged asset with an underlying USDC on Ethereum.
```
[the remark this answers is not in the document]
Comment (marczeller) [reply src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md:14]: changing this and pushing

File: src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      IEngine.Listing({
+        asset: USDCN,
+        assetSymbol: "USDCn",
+        priceFeed: USDCN_PRICE_FEED,
+        rateStrategyParams: Rates.RateStrategyParams({
+          optimalUsageRatio: _bpsToRay(90_00),
+          baseVariableBorrowRate: 0,
+          variableRateSlope1: _bpsToRay(3_50),
+          variableRateSlope2: _bpsToRay(60_00),
+          stableRateSlope1: _bpsToRay(3_50),
```
Comment (sakulstra) [inline src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md:59]:
```suggestion
          stableRateSlope1: _bpsToRay(5_00),
```

File: src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md (hunk 2 above)
Comment (marczeller) [reply src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md]: fixed

**Review by sakulstra — APPROVED** [review]

**Pull request merged**